inference-engine/thirdparty/clDNN/tests/test_cases/convolution_gpu_test.cpp

```diff
@@ -8117,6 +8117,135 @@ static std::string to_string_convolution_all_params(const testing::TestParamInfo
         (padded_input ? "_in_pad" : "");
 }
 
+template <typename InputT, typename WeightsT, typename OutputT>
+class convolution_random_test_fsv4_input : public convolution_random_test_base<InputT, WeightsT, OutputT> {
+public:
+    using parent = convolution_random_test_base<InputT, WeightsT, OutputT>;
+    topology build_topology(const cldnn::engine& engine) override {
+        auto input_lay = layout(this->input_type(), format::b_fs_yx_fsv4, this->input_size());
+        auto wei_lay = layout(this->weights_type(), format::bfyx, this->weights_size());
+
+        auto wei_mem = memory::allocate(engine, wei_lay);
+        auto weights_flat = flatten_4d(format::bfyx, this->_weights);
```

Comment:
Can you explain me why do you prefer using "weights_flat" in on case instead of "wei_flat" as you doing with "wei_mem" line above? I often see in unit-tests such names as "lay, wei, topo, mem...".  I also use part of them. But I amn't sure such abbreviations of words really simplify something or improve code redability. What do you think about? I don't see anywhere recomendations about this...

Reply:
This is just a copy-past from the other test. Renamed to wei_flat for consistency withing the single test.